components/director/internal/domain/systemauth/resolver_test.go

```diff
@@ -162,10 +197,16 @@ func TestResolver_GenericDeleteSystemAuth(t *testing.T) {
 				svc := &automock.OAuth20Service{}
 				return svc
 			},
+			OneTimeTokenFn: func() *automock.OneTimeTokenService {
```

Comment:
You can extract these functions that just return unused mock. See components/director/internal/domain/tenant/resolver_test.go:807 (`unusedTenantConverter`) for example.